test/src/unit-allocator.cpp

```diff
@@ -29,6 +29,10 @@ SOFTWARE.
 
 #include "doctest_compatibility.h"
 
+#if DOCTEST_CLANG && DOCTEST_CLANG >= DOCTEST_COMPILER(3, 6, 0)
+    DOCTEST_CLANG_SUPPRESS_WARNING("-Wkeyword-macro")
```

Comment:
Do you still need this after `JSON_TESTS_PRIVATE` was introduced?

Reply:
I'll have a look if I find time.